django/contrib/gis/static/gis/css/ol3.css

```diff
@@ -0,0 +1,31 @@
+.switch-type {
+  background-repeat: no-repeat;
```

Comment:
I think this should use 4 space indent per https://github.com/django/django/pull/4577. Hopefully we can merge that soon to check JS linting on this patch too.